src/SegmentedMsg.chpl

```diff
@@ -627,16 +626,30 @@ module SegmentedMsg {
             forall (h, h1, h2) in zip(hashes, hash1.a, hash2.a) {
                 (h1,h2) = h:(uint,uint);
             }
-            var repMsg = "created " + st.attrib(name1) + "+created " + st.attrib(name2);
-            smLogger.debug(getModuleName(),getRoutineName(),getLineNumber(),repMsg);
-            return new MsgTuple(repMsg, MsgType.NORMAL);
+            repMsg = "created " + st.attrib(name1) + "+created " + st.attrib(name2);
+        }
+        when ObjType.SEGARRAY {
+            // check to make sure symbols defined
```

Comment:
In what case would it not be defined at this point?

Reply:
if somehow the segments or values arrays were deleted accidentally (we see this happen sometimes when things go out of scope)